language-plutus-core/cost-model-creation/CostModelCreation.hs

```diff
@@ -173,48 +173,71 @@ readModelSplitConst model = (pure . uncurry ModelSplitConst) =<< unsafeReadModel
 addInteger :: MonadR m => (SomeSEXP (Region m)) -> m (CostingFun ModelTwoArguments)
 addInteger cpuModelR = do
   cpuModel <- readModelAddedSizes cpuModelR
-  pure $ CostingFun (ModelTwoArgumentsAddedSizes cpuModel) def
+  -- The worst case is adding e.g. `maxBound :: Int` + `maxBound :: Int`, which increases the memory usage by one.
```

Comment:
... because GMP will need to allocate an additional word of memory in this case.

Reply:
Independent of GMP. I could reference the corresponding GMP functions here, but IMO it's not necessary to make the argument.